# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kaffe/version.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./kaffe/kaffe/version.h.orig	2005-05-18 16:08:45.896346184 -0400
+++ ./kaffe/kaffe/version.h	2005-05-18 16:08:53.322217280 -0400
@@ -10,7 +10,7 @@
 #ifndef KAFFE_KAFFE_VERSION_H
 #define KAFFE_KAFFE_VERSION_H
 
-#define JAVA_VERSION_STRING	"1.1"
+#define JAVA_VERSION_STRING	"1.4"
 
 /*
  * Print copyright notice and simple version info (Java version, 
